contents/handbook/growth/sales/how-we-work.md

```diff
@@ -84,3 +84,35 @@ Other metrics we care about:
 ## Travel to see customers
 
 You are likely to need to travel a lot more than the typical PostHog team member in order to meet customers. Please make sure that you follow our company [travel policy](/handbook/people/spending-money) and act in PostHog's best interests. We trust you to do the right thing here and won't pre-approve your travel plans, but we do keep track of what people are spending and the Ops team will follow up with you if it looks like you are wasting money here. We are not a giant company that pays for fancy flights, accommodation, and meals so please be sensible. 
+
+## Sales team guidelines
+
+### Key Principles
+- Understand our [ICP](https://posthog.com/handbook/who-we-are-building-for): Focus on understanding our users' needs and providing solutions that genuinely help them.
+- Build Credibility: Use PostHog to gain insights into user behavior and demonstrate how our products can solve their problems. Building trust and credibility through helpfulness and relevant information is essential.
+
+### Approach to Existing Accounts
+- Assess Account Activity: Monitor account activity to gauge interest. If an account shows high engagement (e.g., frequent usage, many active users), prioritize a more assertive approach.
```

Comment:
What specifically does this mean? Obviously we should monitor account activity - do you have specific examples of things we've learned that are good predictors of account usage? (e.g. invited lots of users, set up multiple orgs, created insights, viewed replays etc.)

And similarly, what are the flags you've identified?